MetaMask/metamask-extension
# colocate hide-token-confirmation modal styles

**Background**: See the originating PR description at #9147 

**Disclaimer**: this should result in no stylistic or functional changes to the UX.

### Description
Moves the still-alive styles for the hide-token-confirmation to be colocated with the component, as per the examples already set in the `components/app/modals/` folder

### Steps to validate
1. Try to hide a token from the asset page on both a prod build and this PR's build.
2. Compare styles and see that they are the same.
3. Confirm that functionality has not been impacted.

Depends on
- #9147 
- #9148
